usefulUtilities/hiFiCalendar/entityServerScripts/hiFiCalendarServer.js

```diff
@@ -0,0 +1,166 @@
+//  hiFiCalendarServer.js
+//
+//  Created by Mark Brosche on 4/3/2019
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+
+(function() {
+    var that;
+
+    this.remotelyCallable = [
+        "enteredMeetingZone",
+        "leftMeetingZone",
+        "updateSignColor",
+        "showNoScheduledEvents",
+        "addEvent",
+        "clearEventList"
+    ];
+
+
+    this.preload = function(entityID) {
+        that = this;
+        that.entityID = entityID;
+        that.entityProperties = Entities.getEntityProperties(that.entityID, ['id', 'name', 'type', 'parentID']);
+        that.room = {};
+        if (that.entityProperties.name.indexOf("_SCHEDULE") === -1 && 
```

Comment:
do you still need the server script to go on the schedule text entity?

Reply:
also, you don't have to change this in this pr, but for your next PR (as we discussed): please make the different entities in this system have different server scripts on them

Reply:
We could either go with an empty server script on the _SCHEDULE entity or get rid of the room label entity altogether and simply change the background color of the _SCHEDULE entity.  Or we could leave it as is for now and decide in the V2 iteration.

Reply:
the latter sounds fine with me - please make sure you make a note of that for your v2 development!